### PR TITLE
added docs on how to run scripts. Also added some logging during the …

### PR DESCRIPTION
**Pull request by codebuster22**
…scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
 - `/contracts/seed/`- Prime Launch seed module contracts.
 - `/contracts/utils/`- utility contracts.
 
+## Enviornment setup
+
+create ```.env``` file and add following details
+
+```
+PROVIDER =
```
Comment (kinspotting) [inline README.md:20]: `PROVIDER =infura-provider-key`

File: README.md [hunk 2]
```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
...
+MNEMONIC =
```
Comment (kinspotting) [inline README.md:21]: `MNEMONIC =private-key-or-mnemonic`

File: README.md [hunk 3]
```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
...
+```
+
+create ```config.json``` file and add following details
```
Comment (kinspotting) [inline README.md:24]:
```
touch config.json
```

File: README.md [hunk 4]
```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
 - `/contracts/seed/`- Prime Launch seed module contracts.
 - `/contracts/utils/`- utility contracts.
 
+## Enviornment setup
+
+create ```.env``` file and add following details
```
Comment (kinspotting) [inline README.md:17]:
```
touch .env
```

File: README.md [hunk 5]
```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
...
+
+```
+{
+    "SAFE": "gnosis safe address",
+    "ADMIN": "seed admin address",
+    "BENEFICIARY": "prime dao address"
+}
+```
+
+create an empty ```contract-addresses.json``` file
```
Comment (kinspotting) [inline README.md:34]: why do we have to create an empty on if there is one in the repo?

**Review by kinspotting — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (codebuster22) [reply README.md:34]: ohh yes
forgot that we ship the file with the repo :)

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**